models/users.js

```diff
@@ -130,6 +130,25 @@ const fetchUsers = async (query) => {
   try {
     // INFO: default user size set to 100
     // INFO: https://github.com/Real-Dev-Squad/website-backend/pull/873#discussion_r1064229932
+    if (query.member === "true") {
+      const membersArray = [];
+      const snapshot = await userModel.where("roles.member", "==", true).where("roles.archived", "==", false).get();
+      if (!snapshot.empty) {
+        snapshot.forEach((doc) => {
+          membersArray.push({
+            id: doc.id,
+            ...doc.data(),
+            phone: undefined,
+            email: undefined,
+            tokens: undefined,
+            chaincode: undefined,
+          });
+        });
+      }
+      return {
+        allUsers: membersArray,
+      };
+    }
```

Comment:
Please handle this in the bottom part, where users are being fetched. We are just repeating code here.
There is no pagination here, even though user API has pagination.

And instead of having a key where member=true, let's generic field like a filter or role
Where we pass filterBy=MEMBERS, filterBy=AO or role=ARCHIVED, role=NEW, with paginations